nixos/modules/services/web-apps/writefreely.nix

```
@@ -0,0 +1,102 @@
+{config, lib, pkgs, ...}:
+
+with lib;
+
+let
+  cfg = config.services.writefreely;
+  configFile = pkgs.writeText "config.ini" (generators.toINI {} cfg.settings);
+  settingsFormat = pkgs.formats.json {};
+in {
+  options.services.writefreely = {
+    enable = mkEnableOption "WriteFreely, a clean, Markdown-based publishing platform made for writers. Write together, and build a community.";
```

```suggestion
    enable = mkEnableOption "WriteFreely, a Markdown-based publishing platform";
```
we don't usually want to copy the blurb of the packages here, only briefly explain what the thing is and what it does.